src/Control/Carrier/Interpose.hs

```diff
@@ -45,7 +45,7 @@ newtype Handler eff m = Handler (forall x . eff m x -> m x)
 runHandler :: (HFunctor eff, Functor m) => Handler eff m -> eff (ReaderC (Handler eff m) m) a -> m a
 runHandler h@(Handler handler) = handler . hmap (runReader h)
 
-instance (HFunctor eff, Carrier sig m, Member eff sig) => Carrier sig (InterposeC eff m) where
+instance (HFunctor eff, Carrier sig m, Project eff sig) => Carrier sig (InterposeC eff m) where
```

Comment:
This is basically the only thing in the library using `Project`.

I want to 🔥 it so badly, but there’s no way @patrickt will let me 😅

Reply:
_However_, see #220.